README.md

```diff
@@ -14,8 +44,54 @@ To combine lists of events in to a calendar:
 python yaml2ics.py example/test_calendar.yaml example/another_calendar.yaml
 ```
 
+## Syntax
+
+Please see `example/test_calendar.yaml` for a full demo including
+explanations.  Below is a minimal template that shows the basic idea:
+
+```yaml
+name: Calendar Name
+timezone: Europe/Helsinki  # default timezoene for events, optional
+
+events:
+  - summary: The event title
+    begin: 2021-09-21 15:00:00
+    duration:
+	  minutes: 30
+    location: |
+      https://meet.jit.si/example
+    description: |
+	  In this meeting we will ...
+```
+
+## Contributing
+
+Contributions are welcomed! This project is still in active development
+and should be considered beta.
+
+[flit](https://flit.readthedocs.io/en/latest/) is used to manage the project.
+
+```
+pip install flit
+pip install -r requirements.txt  # temporary workaround for ics-py
+flit install -s  # make an editable/inplace installation
+```
```

Comment:
Not necessary for most devs to know; should we put this somewhere else, maybe under "making a release" or similar?

Reply:
Not sure, this part is needed for devs to modify the source and run tests properly. Without this you would need to manually install the dependencies, set the path, etc.

Reply:
I think `pip install -e .` should work perfectly fine?

Reply:
ah right! I forgot it's not poetry, even one of the reasons I went with this 😅